Wolverine.Weather.Infrastructure/Dtos/WeatherForecastDto.cs

```diff
@@ -6,5 +6,6 @@ public class WeatherForecastDto
         public DateTime DateAndTime { get; set; }
         public int TemperatureC { get; set; }
         public string? Summary { get; set; }
+        public Guid ExternalId { get; set; } //Homework: get this External ID returned all the way to the front. Ex. Model on Domain,
```

Comment:
Probably go ahead and get rid of this comment here

Reply:
Removed